README.md

```diff
@@ -22,7 +22,24 @@ implementation 'org.opensearch.client:opensearch-rest-high-level-client:1.1.0'
 
 ## Python
 
-// Coming Soon...
+Testing shows that this is a four (4) line change. Change the `import` statement, and the Python client for ES (Elasticsearch) and OS (OpenSearch).
+
+Changes related to `import`:
+
+```python
+- from elasticsearch import Elasticsearch
++ from opensearchpy import OpenSearch
+```
+
+Changes related to clients and arguments:
+
+```python
+- client_against_opensearch = Elasticsearch(OS_SERVICE_URI, use_ssl=True)
++ client_against_opensearch = OpenSearch(OS_SERVICE_URI, use_ssl=True)
+```
+
+See this commit diff as an example change: [python-client-migration/python_client_migration/main.py](https://github.com/aiven/opensearch-migration-examples/commit/78de47151ab9138b07dafb0bdfb94fa18c4b5566)
```

Comment:
Might be worth using the commit from the current set of commits for this PR
```suggestion
See this commit diff as an example change: [python-client-migration/python_client_migration/main.py](https://github.com/aiven/opensearch-migration-examples/pull/8/commits/7bf1b816aeecb29133cd812c19b116c993dfc0d0)
```

Reply:
We are in sync cause I just thought the same right after I pushed o.o

https://github.com/aiven/opensearch-migration-examples/pull/8/commits/13aa0fcd3bb61a53249633a61d81c091b5379bd4

Reply:
The thing is I was force push and this changes the hashes every time haha. Now, should be fine and also it is more clean.